plugins/fileservices/fileservices.cpp

```diff
@@ -3140,3 +3140,56 @@ FILESERVICES_API void FILESERVICES_CALL fsClearExpireDays(ICodeContext * ctx, co
     else
         throw makeStringExceptionV(0, "ClearExpireDays: Could not find logical file %s", lfn.str());
 }
+
+bool getDefaultValue(const char * processName, const char * propertyName, StringBuffer & strDefaultValue)
+{
+    // Returns with true if a default value retrieved and false if not.
+
+    #ifdef _CONTAINERIZED
+        // TBD
+
+    #else
+        Owned<IConstEnvironment> daliEnv = openDaliEnvironment();
+        Owned<IPropertyTree> env = getEnvironmentTree(daliEnv);
+
+        if (env.get())
+        {
+            Owned<IPropertyTreeIterator> processIter = env->getElements(processName);
+            if (processIter->first())
```

Comment:
> Gavin Halliday:
> If these are global defaults, does it make sense to use the values associated with the 1st dfu server?
> 

> Attila Vamos:
> My original thought/implementation was that, but then it raised several questions/implications, therefore I moved them from Instance to Server Process.

They should not live in the 1st DfuServerProcess, there may be multiple (not just multiple Instance's under each DfuServerProcess).
The original idea was to just allow these values to be set globally, not per dfuserver.
These settings should move into a global section of the environment (maybe EnvSettings, or could add new Software/Globals section).

Reply:
Done

Reply:
minor: The above code implies there could be multiple Globals sections, which doesn't make a great deal of sense.
Better to change not to iterate..

Reply:
@AttilaVamos - Not sure if you've missed my previous comments, re. please leave the "Resolve conversation" button for use by the reviewers only:
https://github.com/hpcc-systems/HPCC-Platform/pull/16528#discussion_r975491777
https://github.com/hpcc-systems/HPCC-Platform/pull/16528#discussion_r977883966